### PR TITLE
feat: update bot csharp templates to msi (batch 2)

### PR DESCRIPTION
**Pull request by yukun-dong**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/28584802

### CODE REVIEW

File: templates/csharp/command-and-response/appsettings.Development.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     }
   },
   "AllowedHosts": "*",
-  "BOT_ID": "$botId$",
-  "BOT_PASSWORD": "$bot-password$"
+  "BOT_ID": "",
+  "BOT_PASSWORD": "",
+  "BOT_TYPE": ""
```
Comment (swatDong) [inline templates/csharp/command-and-response/appsettings.Development.json:12]: It seems some templates (this one) change *appsettings.Development.json* but some (ai-bot) not. Expected?

File: templates/csharp/notification-http-trigger-isolated/appsettings.TestTool.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "BOT_ID": "",
-  "BOT_PASSWORD": ""
+  "MicrosoftAppId": "",
```
Comment (swatDong) [inline templates/csharp/notification-http-trigger-isolated/appsettings.TestTool.json:2]: Mis-match with *Config.cs*

File: templates/csharp/ai-assistant-bot/appsettings.json [hunk 3]
```diff
@@ -7,8 +7,10 @@
     }
   },
   "AllowedHosts": "*",
-  "BOT_ID": "$botId$",
-  "BOT_PASSWORD": "$bot-password$",
+  "BOT_ID": "",
+  "BOT_PASSWORD": "",
+  "BOT_TYPE": "",
+  "BOT_TENANT_ID": "",
```
Comment (swatDong) [inline templates/csharp/ai-assistant-bot/appsettings.json:13]: Any change to *appsettings.TestTool.json*?

File: templates/csharp/command-and-response/appsettings.Development.json (hunk 1 above)
Comment (yukun-dong) [reply templates/csharp/command-and-response/appsettings.Development.json:12]: Missed. Let me update.

File: templates/csharp/ai-assistant-bot/appsettings.json (hunk 3 above)
Comment (yukun-dong) [reply templates/csharp/ai-assistant-bot/appsettings.json:13]: No config is needed for running test tool right? If so I think appsettings.TestTool.json can remain unchanged.

**Review by swatDong — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Pull request merged**